package.json

```diff
@@ -7,12 +7,17 @@
     "url": "git+https://github.com/edx/brand-edx.org.git"
   },
   "author": "edX",
-  "license": "UNLICENSED",
+  "license": "GPL-3.0-or-later",
   "bugs": {
     "url": "https://github.com/edx/brand-edx.org/issues"
   },
   "homepage": "https://github.com/edx/brand-edx.org#readme",
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@edx/paragon": "^21.0.0-alpha.1",
+    "chroma-js": "^2.4.2",
+    "style-dictionary": "^3.7.1"
```

Comment:
[clarification] Brings `style-dictionary` and the `build-tokens.js` script into the theme repositories so they can utilize the same tooling. If we adopt this approach, we may be able to export these things from `@edx/paragon` itself, considering that's already an installed library as seen above.